src/attendees/attendee.py

```diff
@@ -30,6 +32,12 @@ def __init__(self, fname: str, lname: str, email: str,
             file_url (str): The URL to the certificate file, defaults to None.
             kwargs (dict): All other attributes of the attendee.
         """
+        self.fname = fname
+        self.lname = lname
+        self.email = email
+        self.file_path = file_path
+        self.file_url = file_url
+
         raise NotImplementedError
 
     # TODO: Write getters for fname, lname, email, file_path, and file_url
```

Comment:
This still needs to be done. Use the `@property` and `@x.setter` decorators to achieve this.